tests/unit/forklift/test_legacy.py

```diff
@@ -480,24 +480,6 @@ def test_fails_invalid_version(self, pyramid_config, pyramid_request, version):
                 "https://packaging.python.org/specifications/core-metadata"
                 " for more information.",
             ),
-            # version errors.
-            (
-                {"metadata_version": "1.2", "name": "example"},
-                "'' is an invalid value for Version. "
-                "Error: This field is required. "
-                "See "
-                "https://packaging.python.org/specifications/core-metadata"
-                " for more information.",
-            ),
-            (
-                {"metadata_version": "1.2", "name": "example", "version": "dog"},
-                "'dog' is an invalid value for Version. "
-                "Error: Start and end with a letter or numeral "
-                "containing only ASCII numeric and '.', '_' and '-'. "
-                "See "
-                "https://packaging.python.org/specifications/core-metadata"
-                " for more information.",
-            ),
```

Comment:
I realized that these were still valid and added them back in a5779ab08fe0e9a175c73fb74fd2fa8bc1901276.